### PR TITLE
CHE-7881: Implement organize import action with advantage of jdt.ls

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Implement organize import action with advantage of jdt.ls

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7881

### CODE REVIEW

File: plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java [hunk 1]
```diff
@@ -519,4 +528,24 @@ public Location findResourcesByFqn(String fqn, int lineNumber) {
...
+            .executeCommand(
+                new ExecuteCommandParams(
+                    Commands.ORGANIZE_IMPORTS_COMMAND, ImmutableList.of(prefixURI(path))));
+
+    try {
+      return gson.fromJson(gson.toJson(result.get(10, TimeUnit.SECONDS)), WorkspaceEdit.class);
+    } catch (JsonSyntaxException | InterruptedException | ExecutionException | TimeoutException e) {
+      throw new JsonRpcException(-27000, e.getMessage());
+    }
+  }
```
Comment (svor) [inline plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java]: I think you can simplify this method via replacing 
```
public WorkspaceEdit organizeImports(String path) {
      return doGetOne(Commands.ORGANIZE_IMPORTS_COMMAND, prefixURI(path), WorkspaceEdit.class);
}
```

**Review by svor — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

**Review by tsmaeder — CHANGES_REQUESTED** [review]
Just one more thing...

**Comment by tolusha** [comment]
Fixed ;)

**Comment by tolusha** [comment]
Fixed

**Pull request merged**